anomalib/models/components/filters/blur.py

```diff
@@ -9,6 +9,18 @@
 from torch.nn import functional as F
 
 
+def compute_kernel_size(sigma_val: float) -> int:
+    """Compute kernel size from sigma value.
+
+    Args:
+        sigma_val (float): Sigma value.
+
+    Returns:
+        int: Kernel size.
+    """
+    return 2 * int(4.0 * sigma_val + 0.5) + 1
```

Comment:
We use the same computation in Padim ([here](https://github.com/openvinotoolkit/anomalib/blob/main/anomalib/models/padim/anomaly_map.py#L27)), and maybe other models as well. Would be good to use this function there as well.